config/config.go

```diff
@@ -545,6 +546,7 @@ func NewConfig() *Config {
 	flag.Int64Var(&cfg.OpenPolicyAgentMaxRequestBodySize, "open-policy-agent-max-request-body-size", openpolicyagent.DefaultMaxRequestBodySize, "Maximum number of bytes from a http request body that are passed as input to the policy")
 	flag.Int64Var(&cfg.OpenPolicyAgentRequestBodyBufferSize, "open-policy-agent-request-body-buffer-size", openpolicyagent.DefaultRequestBodyBufferSize, "Read buffer size for the request body")
 	flag.Int64Var(&cfg.OpenPolicyAgentMaxMemoryBodyParsing, "open-policy-agent-max-memory-body-parsing", openpolicyagent.DefaultMaxMemoryBodyParsing, "Total number of bytes used to parse http request bodies across all requests. Once the limit is met, requests will be rejected.")
+	flag.IntVar(&cfg.OpenPolicyAgentJwtCacheMaxNumEntries, "open-policy-agent-jwt-cache-max-num-entries", openpolicyagent.DefaultJwtCacheMaxNumEntries, "Maximum number of JWTs to cache for OPA JWT filters")
```

Comment:
This flag is very specific. Do we have other built-ins that actually use the value cache. Enabling this also implicitly disables that we can configure the value cache via the OPA config. 

I am wondering if we should do this a bit more generically (like enable central value cache). 

Apart from this here, we could also read the configuration from the opaconfig that is managed in the registry.